paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PromoBadge.kt

```diff
@@ -58,17 +71,21 @@ internal fun PromoBadge(
 }
 
 @Composable
-private fun formatPromoText(text: String): String {
-    val context = LocalContext.current
-    val currentLocale: Locale = if (SDK_INT >= N) {
-        context.resources.configuration.locales[0]
+private fun formatPromoText(
+    text: String,
+    eligible: Boolean,
+): String {
+    return if (eligible) {
+        val context = LocalContext.current
+
+        if (context.isEnglishLanguage) {
+            "Get $text"
+        } else {
+            text
+        }
```

Comment:
No action - I guess this is a backend limitation, but shouldn't we get complete text from backend?

Reply:
We only get the formatted amount from the backend. That’s the way we intended it, since we need this in multiple places with different text (or none) around it.